src/main/java/bio/terra/service/dataset/flight/upgrade/enableSecureMonitoring/SecureMonitoringEnableFlagStep.java

```diff
@@ -10,21 +10,25 @@
 import bio.terra.stairway.StepStatus;
 import java.util.UUID;
 
-public class EnableSecureMonitoringEnableFlagStep implements Step {
+public class SecureMonitoringEnableFlagStep implements Step {
```

Comment:
This class should probably names something more like `SecureMonitoringSetFlagStep`

Reply:
Ah good point!